src/server.h

```diff
@@ -2142,6 +2142,8 @@ struct valkeyServer {
     int cluster_announce_port;                             /* base port to announce on cluster bus. */
     int cluster_announce_tls_port;                         /* TLS port to announce on cluster bus. */
     int cluster_announce_bus_port;                         /* bus port to announce on cluster bus. */
+    int cluster_announce_client_port;                      /* base port for clients to announce on cluster bus. */
```

Comment:
```suggestion
    int cluster_announce_client_port;                      /* TCP port for clients to announce on cluster bus. */
```
Not sure what base means here, but you don't use it for the TLS port.

Reply:
I mirrored the description for the `cluster_announce_port` value. Should I adjust that comment as well to say `TCP port`?

Reply:
No, cluster_announce_port is actually either TCP or TLS depending on various logic, for historical reasons